app/MorningProgramScreen.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from 'react';
+import { Dimensions, ScrollView, StyleSheet, Text, View } from 'react-native';
+import Carousel from 'react-native-reanimated-carousel';
+
+//importing morning program data
+import { MORNING_PROGRAM_INFO, MorningProgram } from './data/MorningProgramInfo';
+import QuestionIcon from './icons/QuestionIcon';
+
+const { width: screenWidth } = Dimensions.get('window');
+
+const MorningProgramScreen: React.FC = () => {
+  const [activeSlide, setActiveSlide] = useState(0);
+  const morningProgramArray = Object.values(MORNING_PROGRAM_INFO);
+
+  const renderItem = ({ item }: { item: MorningProgram }) => (
+    <ScrollView style={styles.item}>
+      <Text style={styles.title}>{item.title}</Text>
+      <Text style={styles.subtitle}> What's Happening: </Text>
+      <Text style={styles.body}> {item.description}</Text>
+      {item.helpInfo && (
+        <View style={styles.standoutText}>
+          <View style={styles.row}>
+            <QuestionIcon />
+            <Text style={styles.helpSubtitle}>How you can help! </Text>
+          </View>
+          <Text style={styles.helpBody}>{item.helpInfo}</Text>
+        </View>
+      )}
+    </ScrollView>
+  );
+
+  return (
+    <View style={styles.container}>
+      <Carousel
+        data={morningProgramArray}
+        renderItem={renderItem}
+        width={screenWidth}
+        height={670}
+        onSnapToItem={(index) => setActiveSlide(index)}
+        loop={false}
+        autoPlay
+        autoPlayInterval={5000}
+      />
+      <View style={styles.paginationContainer}>
+        {morningProgramArray.map((_, index) => (
+          <View
+            key={index}
+            style={[styles.dotStyle, { opacity: index === activeSlide ? 1 : 0.4 }]}
+          />
+        ))}
+      </View>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
```

Comment:
Can we also try putting "Station #1/#2" on a second line?

![image](https://github.com/DISC-NU/books-and-breakfast-app/assets/36573115/78878a3e-6d9d-4bae-975b-8b09b9a2fcea)